### ISSUE TITLE
Cannot install plugin

### ISSUE THREAD

**Issue by StCyr** [PluginsOCSInventory-NG/services#6]
Map.pm should be renamed map.pm

```
support@houdini:/usr/share/ocsinventory-reports/tools$ sudo python3 install_plugin.py
Where is the plugins location [/usr/share/ocsinventory-reports/ocsreports/extensions/]
/usr/share/ocsinventory-reports/extensions/
[0] => services
0
The server is installed on the same server ? [y]/n
y
Where is the server location [/etc/ocsinventory-server]
/etc/ocsinventory/
ERROR: check if map.pm and [pluginName].conf exist
support@houdini:/usr/share/ocsinventory-reports/tools```

**Comment by StCyr** [comment]
The proper solution seems to be to update the install_plugin.py script in ocsreports.

**Comment by StCyr** [comment]
ah it's already solved in version 2.6
